resolver.go

```diff
@@ -393,6 +394,10 @@ func (r *projectResolver) Accounts(_ context.Context, proj *model.Project) ([]*m
 	return adapter.AccountsToAPI(accounts), nil
 }
 
+func (r *projectResolver) UpdatedAt(_ context.Context, proj *model.Project) (string, error) {
+	return proj.UpdatedAt.Format(time.RFC1123Z), nil
```

Comment:
Why is this even needed?

Reply:
This seems like the simplest way to convert and format the timestamp for graphql to handle it as a string.